go/vt/vtctl/workflow/server.go

```diff
@@ -1277,6 +1277,16 @@ func (s *Server) Materialize(ctx context.Context, ms *vtctldatapb.MaterializeSet
 		cells[i] = strings.TrimSpace(cells[i])
 	}
 
+	if ms.IsReference {
```

Comment:
I think that it's worth doing the same kinds of input validations we're doing in `vtctldclient` to better support other clients that are using the `VtctldServer` RPC directly.

Reply:
fixed